MIGRATION.md

```diff
@@ -1,31 +1,53 @@
 # Migration
 
 ## Migrating from v2 to v3
-- Add the modifier class `o-layout--docs` to the root layout element.
- - Replace class `o-layout__main--full-span` with `o-layout__main__full-span` if used.
-- Your project must set its brand:
+
+- Your project must set its brand to the internal brand:
```

Comment:
Should we explain why this is the case?

Reply:
Maybe I could change it to:

> So all your components are styled for the internal brand your project must set the brand variable. Previously `o-layout` did this, but as it affects all components it is the responsibility of your project to choose a brand. Do so like this:

TMI?

Reply:
Not necessarily TMI, but I was thinking more along the lines of 'o-layout is internal brand only because we need it' and then what you said?